### PR TITLE
Support PaddleOCR-VL LoRA target_modules

### PR DESCRIPTION
**Pull request by forBlank**
#### Before submitting

- [x] Lint code. If there are lint issues, please format the code first.

```shell
# Install and register `pre-commit` in the project folder
pip install pre-commit && pre-commit install

# Process previous code files separately
pre-commit run --file XXXX.py
```

- [x] Add test cases into `tests` folder. If there are codecov issues, please add tests cases first.

### PR types

Others

### PR changes

Models

### Description

Setting PaddleOCR-VL LoRA target_modules for LoRA Training
Use CriterionLayer to compute loss

### CODE REVIEW

File: paddleformers/transformers/paddleocr_vl/modeling.py [hunk 1]
```diff
@@ -2096,8 +2095,9 @@ def forward(
             # Upcast to float if we need to compute the loss to avoid potential precision issues
             logits = logits.astype("float32")
             # Shift so that tokens < n predict n
-            shift_logits = logits[..., :-1, :].contiguous()
-            shift_labels = labels[..., 1:].contiguous()
+            # Labels have beed shifted in dataflow so we don't shift again
```
Comment (lugimzzz) [inline paddleformers/transformers/paddleocr_vl/modeling.py]: 算criterion直接模仿ernie45写法https://github.com/PaddlePaddle/PaddleFormers/blob/develop/paddleformers/transformers/ernie4_5/modeling.py#L1000

**Review by lugimzzz — APPROVED** [review]
LGTM
